hw2/02-search.js

```diff
@@ -0,0 +1,58 @@
+const userInput = document.querySelector("#getData");
+userInput.addEventListener("click", handleClick); // grab data on input click
+
+// Search character array for a match and return the results
+function handleClick() {
+  const search = document.getElementById("userInput").value.trim();
+  console.log(search);
```

Comment:
Interesting that you use console.log here. Is this to aid in testing and debugging? If so, I may need to implement this idea in future code

Reply:
It is for testing!  In fact, I think I should have removed all my debugging console.log() before submitting my PR for "clean code" principals.  Not sure.  I'll have to look into that.  But it's my version of getting one thing working before moving onto the next.  I have a tendency to start too big and have a hard time debugging when that happens.